### PR TITLE
rename namespaces to Unity.MLAgents

### PR DESCRIPTION
**Pull request by chriselion**
### Proposed change(s)
Change all namespaces from `MLAgents*` to `Unity.MLAgents*`. This brings us in line with package guidelines.

### Useful links (Github issues, JIRA tickets, ML-Agents forum threads etc.)
https://jira.unity3d.com/browse/MLA-850

### Types of change(s)

- [ ] Bug fix
- [ ] New feature
- [x] Code refactor
- [x] Breaking change
- [ ] Documentation update
- [ ] Other (please describe)

### Checklist
- [ ] Added tests that prove my fix is effective or that my feature works
- [ ] Updated the [changelog](https://github.com/Unity-Technologies/ml-agents/blob/master/com.unity.ml-agents/CHANGELOG.md) (if applicable)
- [ ] Updated the [documentation](https://github.com/Unity-Technologies/ml-agents/tree/master/docs) (if applicable)
- [ ] Updated the [migration guide](https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Migrating.md) (if applicable)

### CODE REVIEW

File: docs/Learning-Environment-Create-New.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ Then, edit the new `RollerAgent` script:
 
 1. In the Unity Project window, double-click the `RollerAgent` script to open it
    in your code editor.
-2. In the editor, add the `using MLAgents;` and `using MLAgents.Sensors` statements and then change the base
+2. In the editor, add the `using Unity.MLAgents;` and `using Unity.MLAgents.Sensors` statements and then change the base
```
Comment (chriselion) [inline docs/Learning-Environment-Create-New.md:117]: @mmattar heads up that this will conflict with some of your doc changes.

**Comment by chriselion** [comment]
Should I hold off on this change until closer to the release branch date? I'm worried we'll get a lot of support issues for people that are using master docs and release 0.15 code (on vice versa).

**Comment by mmattar** [comment]
> Should I hold off on this change until closer to the release branch date? I'm worried we'll get a lot of support issues for people that are using master docs and release 0.15 code (on vice versa).

@chriselion - I think that may be better.

**Comment by chriselion** [comment]
Will redo on release branch.

**Pull request closed without merging**